nannou/src/wgpu/sampler_builder.rs

```diff
@@ -20,6 +20,7 @@ impl<'b> SamplerBuilder {
     pub const DEFAULT_COMPARE: Option<wgpu::CompareFunction> = None;
     pub const DEFAULT_ANISOTROPY_CLAMP: Option<NonZeroU8> = None;
     pub const DEFAULT_LABEL: Option<&'static str> = None;
```

Comment:
A default label is defined for `TextureView`, but not here -- should these be made consistent?

`pub const DEFAULT_LABEL: &'static str = "nannou-texture-view";`